test/errors/incompatibleKinds-brief.good

```diff
@@ -0,0 +1,5 @@
+error in incompatibleKinds.chpl:3: a regular variable cannot be initialized with a type
+error in incompatibleKinds.chpl:4: a type variable cannot be initialized using a regular value
+error in incompatibleKinds.chpl:7: a 'param' cannot be initialized using a non-'param' value
+error in incompatibleKinds.chpl:10: a regular variable cannot be initialized with a type
+error in incompatibleKinds.chpl:13: a regular variable cannot be initialized with a type
```

Comment:
In brief mode, I find myself wondering why some forms use 'with' and some use 'using'.  If there's not a good reason and we wanted to unify them, of the two, I'd probably choose "with".